### PR TITLE
[ko] Update outdated files dev-1.25-ko.1 (M90-M126)

### PR DESCRIPTION
**Pull request by Seo-yul**
[issues #36863](https://github.com/kubernetes/website/issues/36863)
Update outdated contents of 1.25-ko.1 branch

- [x] M90. content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md | 4(+XS) 0(-)
- [x]  M91. content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md | 1(+XS) 1(-)
- [x]  M92. content/en/docs/tasks/job/indexed-parallel-processing-static.md | 1(+XS) 1(-)
- [x]  M93. content/en/docs/tasks/manage-daemon/update-daemon-set.md | 1(+XS) 1(-)
- [x]  M94. content/en/docs/tasks/manage-gpus/scheduling-gpus.md | 2(+XS) 2(-)
- [x]  M95. content/en/docs/tasks/manage-kubernetes-objects/kustomization.md | 3(+XS) 12(-)
- [x]  M96. content/en/docs/tasks/network/validate-dual-stack.md | 11(+S) 11(-)
- [x]  M97. content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md | 2(+XS) 25(-)
- [x]  M98. content/en/docs/tasks/tls/managing-tls-in-a-cluster.md | 1(+XS) 1(-)
- [x]  M99. content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md | 6(+XS) 2(-)
- [x]  M100. content/en/docs/tutorials/hello-minikube.md | 2(+XS) 2(-)
- [x]  M101. content/en/docs/tutorials/security/apparmor.md | 3(+XS) 46(-)
- [x]  M102. content/en/docs/tutorials/services/source-ip.md | 2(+XS) 2(-)
- [x]  M103. content/en/docs/tutorials/stateful-application/basic-stateful-set.md | 10(+S) 10(-)
- [x]  M104. content/en/examples/admin/konnectivity/konnectivity-server.yaml | 1(+XS) 1(-)
- [x]  M105. content/en/examples/admin/logging/two-files-counter-pod-agent-sidecar.yaml | 1(+XS) 1(-)
- [x]  M106. content/en/examples/admin/sched/pod1.yaml | 1(+XS) 1(-)
- [x]  M107. content/en/examples/admin/sched/pod2.yaml | 1(+XS) 1(-)
- [x]  M108. content/en/examples/admin/sched/pod3.yaml | 1(+XS) 1(-)
- [x]  M109. content/en/examples/application/php-apache.yaml | 1(+XS) 1(-)
- [x]  M110. content/en/examples/application/web/web-parallel.yaml | 1(+XS) 1(-)
- [x]  M111. content/en/examples/application/web/web.yaml | 1(+XS) 1(-)
- [x]  M112. content/en/examples/application/zookeeper/zookeeper.yaml | 1(+XS) 1(-)
- [x]  M113. content/en/examples/debug/node-problem-detector-configmap.yaml | 1(+XS) 1(-)
- [x]  M114. content/en/examples/debug/node-problem-detector.yaml | 1(+XS) 1(-)
- [x]  M115. content/en/examples/pods/inject/dapi-envars-container.yaml | 1(+XS) 1(-)
- [x]  M116. content/en/examples/pods/inject/dapi-envars-pod.yaml | 1(+XS) 1(-)
- [x]  M117. content/en/examples/pods/inject/dapi-volume-resources.yaml | 1(+XS) 1(-)
- [x]  M118. content/en/examples/pods/inject/dapi-volume.yaml | 1(+XS) 1(-)
- [x]  M119. content/en/examples/pods/pod-with-affinity-anti-affinity.yaml | 1(+XS) 1(-)
- [x]  M120. content/en/examples/pods/pod-with-node-affinity.yaml | 1(+XS) 1(-)
- [x]  M121. content/en/examples/pods/pod-with-pod-affinity.yaml | 1(+XS) 1(-)
- [x]  M122. content/en/examples/pods/topology-spread-constraints/one-constraint-with-nodeaffinity.yaml | 1(+XS) 1(-)
- [x]  M123. content/en/examples/pods/topology-spread-constraints/one-constraint.yaml | 1(+XS) 1(-)
- [x]  M124. content/en/examples/pods/topology-spread-constraints/two-constraints.yaml | 1(+XS) 1(-)
- [x]  M125. content/en/releases/_index.md | 2(+XS) 2(-)
- [x]  M126. content/en/releases/version-skew-policy.md | 1(+XS) 1(-)

### CODE REVIEW

File: content/ko/docs/tasks/job/automated-tasks-with-cron-jobs.md [hunk 1]
```diff
@@ -135,19 +126,25 @@ kubectl delete cronjob hello
...
+이미 시작된 작업(및 해당 파드)은 변경 없이 계속 실행된다.
+즉, 크론잡(CronJob)은 기존 작업이 계속 실행 중이라면, 작업을 변경하지 _않는다._
 {{< /note >}}
 
 ### 스케줄
 
 `.spec.schedule` 은 `.spec` 의 필수 필드이다.
-이는 해당 잡이 생성되고 실행되는 스케줄 시간으로 `0 * * * *` 또는 `@hourly` 와 같이 [크론](https://ko.wikipedia.org/wiki/Cron) 형식의 문자열을 받아들인다.
+이는 해당 잡이 생성되고 실행되는 스케줄 시간으로 `0 * * * *` 또는 `@hourly` 와 같이
+[크론](https://ko.wikipedia.org/wiki/Cron) 형식의 문자열을 받아들인다.
```
Comment (jihoon-seo) [inline content/ko/docs/tasks/job/automated-tasks-with-cron-jobs.md:147]: [원문]
> It takes a [Cron](https://en.wikipedia.org/wiki/Cron) format string, such as `0 * * * *` or `@hourly`,
as schedule time of its jobs to be created and executed.

조금 수정해 보았습니다. 😊

```suggestion
이 필드는 `0 * * * *` 또는 `@hourly`와 같은 [크론](https://ko.wikipedia.org/wiki/Cron) 형식의 문자열을 받아, 
해당 잡이 생성 및 실행될 스케줄 시간으로 설정한다.
```

File: content/ko/docs/tutorials/security/apparmor.md [hunk 2]
```diff
@@ -120,7 +118,7 @@ gke-test-default-pool-239f5d02-xwux: kubelet is posting ready status. AppArmor e
 
 <!-- lessoncontent -->
 
-## 파드 보안 강화하기 {#securing-a-pod}
+## 파드 보안 강화하기
 
 {{< note >}}
 AppArmor는 현재 베타라서 옵션은 어노테이션 형식으로 지정한다. 일반 사용자 버전이 되면,
```
Comment (jihoon-seo) [inline content/ko/docs/tutorials/security/apparmor.md:124]:
```suggestion
AppArmor는 현재 베타이며, 이 때문에 옵션은 어노테이션 형식으로 지정한다. 일반 사용자 버전이 되면,
```

File: content/ko/docs/tutorials/services/source-ip.md [hunk 3]
```diff
@@ -326,7 +326,7 @@ kubectl get svc loadbalancer -o yaml | grep -i healthCheckNodePort
 모든 노드의 포트를 가르킨다. 이것을 테스트할 수 있다.
```
Comment (jihoon-seo) [inline content/ko/docs/tutorials/services/source-ip.md:326]:
```suggestion
모든 노드의 포트를 가리킨다. 이것을 테스트할 수 있다.
```

**Review by jihoon-seo — COMMENTED** [review]
@Seo-yul 감사합니다!
몇 가지 의견을 달았습니다.
확인하시고 선택적으로 반영해 주시고,
수정 반영 사항이 많다면 squash 하지 않고 커밋을 추가하는 것을 권장드립니다. 😊

File: content/ko/docs/tasks/job/automated-tasks-with-cron-jobs.md [hunk 4]
```diff
@@ -190,8 +190,10 @@ kubectl delete cronjob hello
 명세는 다음의 동시성 정책 중 하나만 지정할 수 있다.
 
 * `Allow`(기본값): 크론 잡은 동시에 실행되는 잡을 허용한다.
-* `Forbid`: 크론 잡은 동시 실행을 허용하지 않는다. 새로운 잡을 실행할 시간이고 이전 잡 실행이 아직 완료되지 않은 경우, 크론 잡은 새로운 잡 실행을 건너뛴다.
-* `Replace`: 새로운 잡을 실행할 시간이고 이전 잡 실행이 아직 완료되지 않은 경우, 크론 잡은 현재 실행 중인 잡 실행을 새로운 잡 실행으로 대체한다.
+* `Forbid`: 크론 잡은 동시 실행을 허용하지 않는다.
+새로운 잡을 실행할 시간이고 이전 잡 실행이 아직 완료되지 않은 경우, 크론 잡은 새로운 잡 실행을 건너뛴다.
+* `Replace`: 새로운 잡을 실행할 시간이고 이전 잡 실행이 아직 완료되지 않은 경우,
+크론 잡은 현재 실행 중인 잡 실행을 새로운 잡 실행으로 대체한다.
```
Comment (Seo-yul) [inline content/ko/docs/tasks/job/automated-tasks-with-cron-jobs.md:197]:
```suggestion
* `Forbid`: 크론 잡은 동시 실행을 허용하지 않는다.
  새로운 잡을 실행할 시간이고 이전 잡 실행이 아직 완료되지 않은 경우, 크론 잡은 새로운 잡 실행을 건너뛴다.
* `Replace`: 새로운 잡을 실행할 시간이고 이전 잡 실행이 아직 완료되지 않은 경우,
  크론 잡은 현재 실행 중인 잡 실행을 새로운 잡 실행으로 대체한다.

```

**Review by jihoon-seo — COMMENTED** [review]
@Seo-yul 고생 많으셨습니다 감사합니다!
이제 squash 해 주시면 될 것 같습니다. 😊
/lgtm

**Comment by Seo-yul** [comment]
> @Seo-yul 고생 많으셨습니다 감사합니다! 이제 squash 해 주시면 될 것 같습니다. 😊 /lgtm

확인 감사합니다! 진행하였습니다.

**Pull request merged**